CodeEdit/Features/WindowCommands/FileCommands.swift

```diff
@@ -72,4 +75,79 @@ struct FileCommands: Commands {
             .keyboardShortcut("s")
         }
     }
+
+    // TODO: This logic needs to become a command, not a function bound to the MenuBar
```

Comment:
Where would functions like this be stored? If we stored everything in a command, there would be a lot of random files and functions floating around. If we were to do this we would need to come up with a way to neatly organize them all. This might coincide with how Settings is organized.